src/screens/error/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { Typography } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import { Button } from '../../components/button';
+import Copyright from '../../components/copyright';
+import ErrorAnimation from '../../components/errorAnimation';
+import Header from '../../components/Header';
+import useHeader from '../../components/Header/hooks/useHeader';
+import {
+  Container,
+  ContainerCopyright,
+  ContainerLogo,
+  ContainerMessage,
+  OutsideContainer,
+} from './styles';
+
+const Error = () => {
+  const navigate = useNavigate();
+  const { handleLogoClick } = useHeader();
+  return (
+    <>
+      <OutsideContainer>
+        <Container>
+          <ContainerLogo>
+            <Header showLogin={false} />
+          </ContainerLogo>
+          <ContainerMessage>
+            <ErrorAnimation />
+            <Typography variant="h3">
+              Desculpe, não encontramos a página que você procura
+            </Typography>
+            <Typography variant="h6">
```

Comment:
Mostrei pro chris e ele achou que aqui fica melhor um `body1`. Faz essa mudança, pf?
```suggestion
            <Typography variant="body1">
```